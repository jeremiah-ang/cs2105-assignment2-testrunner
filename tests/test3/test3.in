hello1
bye1
see you again!1
/send huge_file.zip
hello2
bye2
see you again!2
