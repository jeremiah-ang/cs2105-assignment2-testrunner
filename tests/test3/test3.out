hello1
bye1
see you again!1
hello2
bye2
see you again!2
